dolphinscheduler-remote/src/main/java/org/apache/dolphinscheduler/remote/handler/NettyClientHandler.java

```diff
@@ -175,4 +183,16 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
         ctx.channel().close();
     }
 
+    @Override
+    public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exception {
+        if (evt instanceof IdleStateEvent) {
+            Command heartBeat = new Command();
```

Comment:
It's better if we distinguish the READER_IDLE and WRITER_IDLE event

Reply:
As far as heartbeat detection is concerned, this seems to be redundant. Regardless of the read and write events, they are all the same events. What do you think?